pages/rdm_cycle/sharing.md

```diff
@@ -1,40 +1,47 @@
-  * Research funders, institutions and reputable journals/publishers with data sharing mandates will normally allow you to opt out of their open data requirements only for legitimate reasons (see below).
-  * Additional reasons to share your datasets:
-    - [Ten reasons to share your data.](https://www.natureindex.com/news-blog/ten-reasons-to-share-your-data)
-    - [Ask not what you can do for open data; ask what open data can do for you.](http://blogs.nature.com/naturejobs/2017/06/19/ask-not-what-you-can-do-for-open-data-ask-what-open-data-can-do-for-you/)
-* Good reasons to close or restrict access to your datasets.
-  - Ethical issues.
-  - Personal data.
-  - Sensitive data that could potentially cause harm (e.g. to endangered species, vulnerable sites or groups, public health, national security…) if made public. Some personal data are considered sensitive personal data.
-  - Legal or contractual issues:
-    - Confidential data (you have a duty or agreed to keep the data confidential).
-    - Third-party data (data are not generated in the course of your own research project, but are supplied to you by another party).  
-    - Copyright and/or database right data.
-    - Data with potential economic/commercial value.
-
-If you want to close or restrict access to your datasets, you must explain your ethical and/or legal issues in the Data Management Plan. You can ask help to the Legal Team, Tech Transfer Office or Data Protection Officer of your institute.
-
-* For finding appropriate repositories by type of data, there are extensive lists and selection tools in the following locations:
-  - For Biomolecular Data in Life Science use one of the [ELIXIR Deposition Databases](https://elixir-europe.org/platforms/data/elixir-deposition-databases), or use the [EBI Guidance Wizard](https://www.ebi.ac.uk/submission/)
-  - [FAIRsharing](https://fairsharing.org/)
-  - [Re3data](https://re3data.org/)
-  - [Repositories recommended by Nature publishing group](https://www.nature.com/sdata/policies/repositories)
-  - For the sharing of human-subject phenotype/genotype data there are well-established repositories ([EGA](https://ega-archive.org/), [dbGAP](https://www.ncbi.nlm.nih.gov/gap/), [JGA](https://www.ddbj.nig.ac.jp/jga/index-e.html)). The choice will depend on the conditions under which data has been donated and the legislative/geographic boundaries for the research project/consortium. There are also community-specific data hubs that may implement a controlled-access model e.g. ORPHANET, MJFF.
+
+* Take Data Sharing into consideration already at the [Planning stage](planning). Don't leave it to the last stage. 
+* Making the data as FAIR as possible will ensure that the most value can be made out of it in the future.
```

Comment:
You might want to add a separate point about data usage licenses (FAIR principle R.1.1), since this can be critical for making shared data reusable.

Reply:
We can link to `pages/your_problem/data_licence.md` for licenses.

Reply:
@nsoranzo on a small note, we decided this week to rename  `data_licence.md` to `licensing.md`

Reply:
> on a small note, we decided this week to rename `data_licence.md` to `licensing.md`

Fine by me!